packages/client/e2e/_utils/page.ts

```diff
@@ -82,11 +70,29 @@ export async function setupAndGetUserPage({
 
     await initPage(page);
 
+    await PerformSignUp(page);
+
+    await initPage(page);
+
+    const storageState = await context.storageState();
+
+    const userCredentialsLocalStorage = storageState.origins
+        .slice(-1)[0]
+        .localStorage.find((el) => el.name === 'USER_CREDENTIALS');
+
+    invariant(
+        userCredentialsLocalStorage !== undefined,
+        'could not retrieve user credentials from local storage',
+    );
+    const { userID, userNickname } = UserCredentials.parse(
+        JSON.parse(userCredentialsLocalStorage.value),
+    );
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice, like that tests are aware of the information of the currently logged in user 👌.